README.md

```diff
@@ -0,0 +1,12 @@
+Mender Test Containers
+==============================================
+
+This repository contains docker image definitions needed to continously build and test products for the Mender company.
+
+The images in here are used in the following situations:
+
+| docker tag | folder | usage example |
+|---|---|---|
+| raspbian_latest | docker | [this repo](https://github.com/mendersoftware/mender-test-containers/blob/master/container_props.py) for use in [mender-binary-delta](https://github.com/mendersoftware/mender-binary-delta/blob/master/.gitmodules) |
```

Comment:
Actually, it is for use in `mender-dist-packages`. `mender-binary-delta` does import the module but only to use qemu images.

I'll amend this myself later